### PR TITLE
Added labeller function and updated dependencies

### PR DESCRIPTION
**Pull request by the-Hull**
Dear devs,

thanks for the effort you've put into this package / data set. 
In trying to re-build the package with a unit labelling function, I noticed issues with tests (`storr::hash_object()`) causing my builds to fail. 

I updated the dependencies in (including a md5 hash) in the tests, and added the labeller function (with documentation and corresponding tests) - perhaps somebody will find it useful.

Cheers and stay healthy,
Alex

### CODE REVIEW

**Comment by dfalster** [comment]
Hi @the-Hull ! Thanks heaps for this. Really appreciate your input and suggestions for new features. I'll be interested to see the labeller function.  I'll check this out in detail hopefully next week. But just wanted to say a quick thank you in the meantime.

**Comment by the-Hull** [comment]
No worries! Quick example:

```
libary(baad.data)

bd <- baad_data()

xlab <- baad_labeller(bd$dictionary, "r.st", as_expression = TRUE)
ylab <- baad_labeller(bd$dictionary, "h.t", as_expression = TRUE)

plot(bd$data$r.st, bd$data$h.t, xlab = xlab, ylab = ylab)
```

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # baad.data
 
-[![Build Status](https://travis-ci.org/traitecoevo/baad.data.png?branch=master)](https://travis-ci.org/traitecoevo/baad.data)
+[![Build Status](https://travis-ci.org/the-Hull/baad.data.svg?branch=master)](https://travis-ci.org/the-Hull/baad.data)
```
Comment (dfalster) [inline README.md]: Can we change this back to traitecoevo?

**Review by dfalster — CHANGES_REQUESTED** [review]
Thanks for the updates @the-Hull. These additions are very helpful and I'd like to merge them in.

Can I request two changes first?

1. Fix travis link to be traitecoevo
2. Update the pull request to include 1-2 commits only? currently there's 13 commits - most of which are just intermediaries. Either I merge squash your work into a single commit, or you update the PR to include only 1-2 commits and I merge that without squashing.

File: README.md (hunk 1 above)
Comment (the-Hull) [reply README.md]: Sorry for the late follow up @dfalster 
I'll get to this this week, most likely! 
Cheers!

Comment (the-Hull) [reply README.md]: Sorry for the delay - I will try to squash the commits!

**Pull request closed without merging**